executor/standalone/standalone_test.go

```diff
@@ -1304,3 +1300,50 @@ func TestBasicMultiContainer(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestMultiContainerDoesPlatformFirst(t *testing.T) {
+	wrapTestStandalone(t)
+	skipIfNotPod(t)
+
+	// And for the main container, we use pgrep to ensure that our *user* sentinel container
+	// is in fact running along side us.
+	// It will only work if the user sentinel sidecar is seen running by the time we start.
+	testEntrypointOld := "pgrep -fx '/bin/sleep 430'"
+	// The main container and the user-sentinel are both 'user' containers,
+	// So we want the main container to waid just a little bit for the user-sentinel
+	// to come up, report back if the platform-sentinel is running or not, and then continue
+	testEntrypointOld = `/bin/sh -c "sleep 3;` + testEntrypointOld + `"`
+
+	ji := &JobInput{
+		ImageName:  busybox.name,
+		Version:    busybox.tag,
+		UsePodSpec: shouldUsePodspecInTest,
+		// This sentinel container is a second process we can look out
+		// for, in order to detect if multi-container workloads are setup
+		ExtraContainers: []corev1.Container{
+			// This first one is a platform sidecar with a special sleep, this
+			// *should* run first if the code is correct.
+			{
+				Name:    "platform-sentinel",
+				Image:   busybox.name + `:` + busybox.tag,
+				Command: []string{"/bin/sh", "-c"},
+				Args:    []string{"/bin/sleep 420"},
+			},
+			// Second is a user sidecar, it *should* run second. If it sees the platform-sentinel
+			// running, only then will it keep running with its own sleep
+			{
+				Name:    "user-sentinel",
+				Image:   busybox.name + `:` + busybox.tag,
+				Command: []string{"/bin/sh", "-c"},
+				Args:    []string{"pgrep -fx '/bin/sleep 420' && /bin/sleep 430"},
+			},
+		},
+		ExtraAnnotations: map[string]string{
+			"type.pod.netflix.com/platform-sentinel": "sidecar",
```

Comment:
Might be nice to directly use the annotation name from `podCommon`?

Reply:
Oh yea